quidem-ut/src/test/quidem/org.apache.druid.quidem.QTest/qaUnnest/array_sql_col_datatype_array.std.iq

```diff
@@ -0,0 +1,805 @@
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[3]' RETURNING varchar array)) AS u(c)
+GROUP BY c;
++----+
+| c  |
++----+
+| S1 |
+| S2 |
+|    |
++----+
+(3 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A19_B1 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[4]' RETURNING varchar array)) AS u(c);
++---+
+| c |
++---+
+|   |
+|   |
+|   |
+|   |
+|   |
+|   |
+|   |
+|   |
+|   |
++---+
+(9 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A19_B2 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[4]' RETURNING varchar array)) AS u(c)
+GROUP BY c;
++---+
+| c |
++---+
+|   |
++---+
+(1 row)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A20_B1 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[5]' RETURNING varchar array)) AS u(c);
++---+
+| c |
++---+
++---+
+(0 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A20_B2 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[5]' RETURNING varchar array)) AS u(c)
+GROUP BY c;
++---+
+| c |
++---+
++---+
+(0 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A21_B1 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[6]' RETURNING varchar array)) AS u(c);
++------+
+| c    |
++------+
+| 0.1  |
+| 0.1  |
+| 0.1  |
+| 1    |
+| 1    |
+| 1    |
+| S1   |
+| S1   |
+| S1   |
+| true |
+| true |
+| true |
+|      |
+|      |
+|      |
++------+
+(15 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# TESTCASE: test_col_datatype_array TEST_ID: A21_B2 TYPE: POSITIVE TEST
+#-------------------------------------------------------------------------
+SELECT c
+FROM test_unnest,
+     unnest(json_value(a_nested, '$[6]' RETURNING varchar array)) AS u(c)
+GROUP BY c;
++------+
+| c    |
++------+
+| 0.1  |
+| 1    |
+| S1   |
+| true |
+|      |
++------+
+(5 rows)
+
+!ok
+
+#-------------------------------------------------------------------------
+# Total query count 43 Positive tests: 37 Negative tests: 6
```

Comment:
I don't think these headers are usefull - instead they could cause confusion:
what happens if tomorrow one more `NEGATIVE` tests will be supported...should these numbers be adjusted as well?

Reply:
This is for accounting purpose only.  I can remove them.

Reply:
Addressed in https://github.com/apache/druid/pull/17861/commits/e2df8ea12b4b27df86b5fa28668eb38f738cc679